docs/nodes/validator-node-handling.md

```diff
@@ -180,27 +181,27 @@ If you don't use `kms` the private key of your validator is saved in `priv_valid
    systemctl stop commercionetworkd
    systemctl disable commercionetworkd
    ```
-1. Sync again your new node `data` folder
+5. Sync again your new node `data` folder
```

Comment:
Note: It is good that you want to respect the order of the numbers, but markdown renders the right sequence, so getting them right doesn't make a big difference